tasks/mock-github

```diff
@@ -39,6 +40,10 @@ class Handler(MockHandler):
             self.send_error(404, 'Mock Not Found: ' + self.path)
 
     def do_POST(self):
+        if log:
+            print("POST", self.path, self.rfile.read1().decode(), file=log)
```

Comment:
...why not (also) GET?

Reply:
This is mostly meant to validate actions. GET requests don't change the state, they are already sufficiently defined in terms of which paths `do_GET` handles in the test, and they are also somewhat of an implementation detail. I can add them to the log of course, but I wouldn't have anything to assert on them. For human inspection it's nice though, I agree.

Added.